tests/testthat/helper-integration-tests-setup.R

```diff
@@ -259,24 +259,20 @@ setup_and_run_FIMS <- function(iter_id,
 #' # The `result` list contains components such as `report`, `sdr_report`, and `obj`.
 #'
 #' validate_fims(
-#'  report = result$report,
-#'  sdr = TMB::sdreport(result$obj),
-#'  sdr_report = result$sdr_report,
-#'  om_input = om_input_list[[1]],
-#'  om_output = om_output_list[[1]],
-#'  em_input = em_input_list[[1]]
+#'   report = result$report,
+#'   sdr = TMB::sdreport(result$obj),
+#'   sdr_report = result$sdr_report,
+#'   om_input = om_input_list[[1]],
+#'   om_output = om_output_list[[1]],
+#'   em_input = em_input_list[[1]]
 #' )
-#'
-#' #' Note: Some sections of the function have commented-out code for additional checks
-#' that are not currently active.
```

Comment:
These could have been left in if you want to use
```
#' # Note: some sections of the function have commented-out code for additional checks
#' # that are not currently active.
```
But, I am fine with it how it is without the comment as well.

This leads me to a bigger picture question that I have talked to @k-doering-NOAA about regarding should the documentation GitHub action be run during the PR phase rather than after merging into main. We thought it was not worth implementing at this current point in time but if anyone has ideas for how to ensure that clean up is part of the PR rather than after merging into main that would be amazing ⭐!

Reply:
I think that running doc-and-style on PRs makes a lot of sense. It gives PR authors better feedback on their code styling and gives the reviewers a chance to look over changes to the .Rd files and other documentation updated by doc-and-style.

I just implemented that for r4ss (via https://github.com/r4ss/r4ss/commit/5be028c1825610d3c61be85b2691afd1735b75f9) and posted an issue about it here: https://github.com/nmfs-fish-tools/ghactions4r/issues/135. There may be some redundant runs of the workflows but I don't think that's a big deal for something that runs pretty fast.

Reply:
I like the idea of doing cleanup during the pull request process. I found a [use case](https://github.com/peter-evans/create-pull-request/blob/main/docs/examples.md#autopep8) in the create-pull-request repository that might work for us.

Reply:
I've created a new issue https://github.com/NOAA-FIMS/FIMS/issues/660 to continue the discussion. I will close this PR and fixe the issue on the main branch.